src/ingestion/models/ingestionManager.ts

```diff
@@ -206,7 +221,9 @@ export class IngestionManager {
         msg: message,
         logCtx: logCtx,
       });
-      throw new ConflictError(message);
+      const error = new ConflictError(message);
+      trace.getActiveSpan()?.recordException(error);
```

Comment:
make sure `recordException` hold the correct status code - if its does not add statusCode

Reply:
done - set status as an attribute